### DISCUSSION TITLE
General Question for PYMOO

### DISCUSSION THREAD

**Discussion by wuyoscar** [anyoptimization/pymoo#194] [Q&A] [answered]
Hi there. Hopy you are well.
I have been used pymoo pacakge for a while. It is really helpful to my work. Thanks
Also, i have some questions.

For example, first we initial problem, algorithmn. And then use `minize` function to find  best soluiton (res.X) and 
corresponding objective value (res.F). They are optimal solution set (non-dominated and feasible). Is that right? If so, is there any function in pymoo, which can find paretofront given res.F ?  

best regards

**Comment by blankjul** [comment, marked as answer]
I am not exactly sure what you are asking. `res.F` is the output of an optimization run which shall be as close as possible to `pf`. The `pf` is mathematically derived for test problems, but for real-world problems it is unknown.

Your question might also be related to solving an `inverse problem` which might be worth reading about.
